compiler/passes/codegen.cpp

```diff
@@ -651,9 +684,9 @@ static void codegen_header() {
   //
   forv_Vec(FnSymbol, fn, gFnSymbols) {
     legalizeName(fn);
-    functions.add(fn);
+    functions.push_back(fn);
   }
-  qsort(functions.v, functions.n, sizeof(functions.v[0]), compareSymbol);
+  std::sort(functions.begin(), functions.end(), compareSymbol2);
```

Comment:
Does the comparison function pointer get automatically promoted to the Comparison functor type? or does it "just work" after generic instantiation?  I have always gone through the rigamarole of packaging the comparison operator in a functor class, so I was unaware that one could skip that syntax.